src/uct/cuda/cuda_ipc/cuda_ipc_ep.c

```diff
@@ -38,10 +44,55 @@ UCS_CLASS_DEFINE_DELETE_FUNC(uct_cuda_ipc_ep_t, uct_ep_t);
 #define uct_cuda_ipc_trace_data(_addr, _rkey, _fmt, ...)     \
     ucs_trace_data(_fmt " to %"PRIx64"(%+ld)", ## __VA_ARGS__, (_addr), (_rkey))
 
+void *uct_cuda_ipc_ep_attach_rem_seg(uct_cuda_ipc_ep_t *ep,
+                                     uct_cuda_ipc_iface_t *iface,
+                                     uct_cuda_ipc_key_t *rkey)
+{
+    unsigned int cuda_ipc_mh_flags = CU_IPC_MEM_LAZY_ENABLE_PEER_ACCESS;
```

Comment:
may be  you can directly use flag in cuIpcOpenMemHandle

Reply:
maybe you can align arguments to "UCT_CUDADRV_FUNC(".  but i will this to @yosefe @brminich review